manifests/claudie/kustomization.yaml

```diff
@@ -1,38 +1,40 @@
 namespace: claudie
 resources:
-- ns.yaml
-- context-box.yaml
-- terraformer.yaml
-- ansibler.yaml
-- scheduler.yaml
-- builder.yaml
-- kube-eleven.yaml
-- frontend.yaml
-- kuber.yaml
-- minio
-- mongo
-- dynamo
-- network-policy.yaml
+  - ns.yaml
```

Comment:
❤️

Reply:
The `kustomize edit ` will revert it back anyway :smile:

Reply:
Really? When running locally, I did not have issues with it 🤔 Did it started working like this when I bumped the version to `4.5.6`?